<launch> 
    <node name="oximeter" pkg="oximeter" type="oximeter" output="screen" />

    <!-- Defines the percentages to consider low, moderate or high risk -->
    <param name="lowrisk" value="0,20" />
    <param name="midrisk" value="21,65" />
    <param name="highrisk" value="66,100" />

    <!-- markov chain -->
    <param name="state0" value="0,0,100,0,0" />
    <param name="state1" value="0,0,100,0,0" />
    <param name="state2" value="0,0,100,0,0" />
    <param name="state2" value="0,0,0,0,0" />
    <param name="state2" value="0,0,0,0,0" />

    <!-- risk values -->
    <param name="LowRisk" value="0,54.99" />
    <param name="MidRisk1" value="55,64.99" />
    <param name="HighRisk1" value="65,100" />

    <!-- accuracy in percentage -->
    <param name="data_accuracy" value="90" type="double" />
    <param name="comm_accuracy" value="100" type="double" />

    <!-- persistence -->
    <param name="persist" value="true" type="bool" />
    <param name="path" value="oximeter_output.csv" type="str" />

</launch>